angular-legacy/dmp/app/dmp-form.component.ts

```diff
@@ -153,14 +153,20 @@ export class DmpFormComponent extends LoadableComponent {
       this.initSubs.unsubscribe();
       translationService.isReady(tService => {
         this.fieldMap = {_rootComp:this};
+        console.log(this.fieldMap);
+        console.log('================= fieldMap');
```

Comment:
This code is going to be ported soon and we have a logging service with more control that can probably be used but for the meantime best to remove these debug logging statements (and the ones below)